app/mailers/project_status_change_mailer.rb

```diff
@@ -0,0 +1,17 @@
+# Sends emails to notify project team members when their project status has changed
+# @see http://localhost:3000/rails/mailers/project_status_change_mailer/
+class ProjectStatusChangeMailer < ApplicationMailer
+  def approved(member:, approval:)
+    mail(to: member.email) do |format|
+      format.html { render locals: { approval: approval, member: member } }
+      format.text { render locals: { approval: approval, member: member } }
```

Comment:
This is because I didn't want to define `@vars` all over the place; but I'm fine to do that since it is more conventional.